op-challenger/fault/responder.go

```diff
@@ -72,6 +77,16 @@ func (r *faultResponder) BuildTx(ctx context.Context, response Claim) ([]byte, e
 	}
 }
 
+// Resolve executes a resolve transaction to resolve a fault dispute game.
+func (r *faultResponder) Resolve(ctx context.Context) error {
```

Comment:
yeah this is what I was expecting instead of https://github.com/ethereum-optimism/optimism/pull/6349/

Adding this as a conversation to prevent auto merging but I think this is what we want and we just close https://github.com/ethereum-optimism/optimism/pull/6349 - if that's what you had in mind go ahead and merge this, otherwise let's talk because I'm obviously missing something. :)